win/CS/HandBrakeWPF/ViewModels/OptionsViewModel.cs

```diff
@@ -52,10 +52,10 @@ public class OptionsViewModel : ViewModelBase, IOptionsViewModel
         private bool changeToTitleCase;
         private bool checkForUpdates;
         private UpdateCheck checkForUpdatesFrequency;
-        private bool clearOldOlgs;

```

Comment:
This garbled boolean makes it hard to understand what it's for.
I'm not certain I'm right.